app/controllers/profile_controller.rb

```diff
@@ -1,4 +1,8 @@
+# frozen_string_literal: true
```

Comment:
I think for consistency we should probably take this out as we have done in every other file. This doesn't arrive fully until [v3 anyway](https://bugs.ruby-lang.org/issues/8976#note-30).

Reply:
Removed